cmd/grpcurl/grpcurl.go

```diff
@@ -235,7 +235,7 @@ func main() {
 				fail(err, "Failed to configure transport credentials")
 			}
 			if *serverName != "" {
-				creds.OverrideServerName(*serverName)
+				_ = creds.OverrideServerName(*serverName)
```

Comment:
warn?

Reply:
Doh! I was mechanically ignoring errors, but this is a real violation that needs to be fixed!